hera_cal/lstbin.py

```diff
@@ -675,7 +677,7 @@ def lst_bin_files(data_files, input_cals=None, dlst=None, verbose=True, ntimes_p
                     file_list.append(data_files[j][k])
                     nightly_data_list.append(data)  # this is data
                     nightly_flgs_list.append(flags)  # this is flgs
-                    nightly_lst_list.append(larr[tinds])  # this is lsts
+                    nightly_lst_list.append(data.lsts)  # this is lsts
```

Comment:
can you remind me if this was a necessary change, or a cosmetic change? I know that a unique copy of the lsts are attached to the data DataContainer, but I'm wondering why larr[tinds] doesn't work also. My concern is that it is very clear what `larr[tinds]` is, but with the change we are now tying ourselves to be dependent on however data.lsts is defined, which I'm wary may for some reason change without us knowing in the future. 

lmk thanks

Reply:
I think this is necessary. We were previously not passing unwrapped LSTs to the lstbinner with `larr`. Since `larr` is now getting modified, this change prevents the modifed version from escaping outside this function. I could just copy larr before modifying and pass the copy instead.